plumber/api/importer.py

```diff
@@ -49,68 +45,171 @@ class ParallelImportBuilder:
     def __init__(
         self,
         file_system: GameFileSystem,
+        # General settings
```

Comment:
These are general settings only as an implementation detail. In the API surface, these should be grouped under respective categories, e.g. import lights - sky equi height under vmf, scale - remove animations under MDL, and import unknown entities under vmf too. Import materials can be put under material settings.